instrumentation/instalambda/triggers.go

```diff
@@ -71,3 +89,24 @@ func detectTriggerEventType(payload []byte) triggerEventType {
 		return unknownEventType
 	}
 }
+
+func areTracingHeadersInTheCustomContext(lcc lambdacontext.ClientContext) bool {
+	if lcc.Custom == nil {
+		return false
+	}
+
+	normalizedCustomKeys := make(map[string]string, len(lcc.Custom))
+
+	for k := range lcc.Custom {
+		normalizedCustomKeys[strings.ToUpper(k)] = k
```

Comment:
Trace context keys are case-sensitive. I believe we can skip normalization here.

Reply:
I will change it